rbi/stdlib/openssl.rbi

```diff
@@ -7404,6 +7404,16 @@ class OpenSSL::SSL::SSLContext
   # Server sessions are added to the session cache
   SESSION_CACHE_SERVER = ::T.let(nil, ::T.untyped)
 
+  sig do
+    params(
+      certificate: ::OpenSSL::X509::Certificate,
+      pkey: ::OpenSSL::PKey::PKey,
+      extra_certs: ::T.nilable(T::Array[::OpenSSL::X509::Certificate])
+    )
+    .returns(::OpenSSL::SSL::SSLContext)
+  end
+  def add_certificate(certificate, pkey, extra_certs = nil);
```

Comment:
```suggestion
  def add_certificate(certificate, pkey, extra_certs = nil); end
```

Reply:
cc @froydnj I think this is why the Sorbet build failed, because there was a syntax error in the payload.

Reply:
Good catch! This is now fixed

Reply:
Thanks for pointing that point.  Feels like a terrible failure mode. :(